src/SFA.DAS.EmployerFinance.Web/Helpers/AsyncHelper.cs

```diff
@@ -0,0 +1,33 @@
+using System.Globalization;
+
+namespace SFA.DAS.EmployerFinance.Web.Helpers;
+
+public static class AsyncHelper
+{
+    private static readonly TaskFactory _taskFactory = new(CancellationToken.None,
+        TaskCreationOptions.None, TaskContinuationOptions.None, TaskScheduler.Default);
+
+    public static TResult RunSync<TResult>(Func<Task<TResult>> func)
+    {
+        var cultureUi = CultureInfo.CurrentUICulture;
+        var culture = CultureInfo.CurrentCulture;
+        return _taskFactory.StartNew(() =>
+        {
+            Thread.CurrentThread.CurrentCulture = culture;
+            Thread.CurrentThread.CurrentUICulture = cultureUi;
+            return func();
+        }).Unwrap().GetAwaiter().GetResult();
+    }
+
+    public static void RunSync(Func<Task> func)
+    {
+        var cultureUi = CultureInfo.CurrentUICulture;
+        var culture = CultureInfo.CurrentCulture;
+        _taskFactory.StartNew(() =>
+        {
+            Thread.CurrentThread.CurrentCulture = culture;
+            Thread.CurrentThread.CurrentUICulture = cultureUi;
+            return func();
+        }).Unwrap().GetAwaiter().GetResult();
+    }
+} 
```

Comment:
This worries me a little, it appears top be a duplicate of another file, but this instance doesn't appear to be used. 

calling GetResult() does raise red flags to me